src/test/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudServiceTest.kt

```diff
@@ -74,11 +88,52 @@ class CosmasGoogleCloudServiceTest {
         assertNotNull(stream2.error)
     }
 
-    private fun getFileFromService(version: Int, fileId: String = "0", projectId: String = "0"): ByteString {
+    @Test
+    fun addSecondVersionAndCheckListVersions() {
+        val fakeStorage: Storage = mock(Storage::class.java)
+        val blob1 = getMockedBlob("ver1".toByteArray(), 1)
+        val blob2 = getMockedBlob("ver2".toByteArray(), 2)
+        val fakePage: Page<Blob> = mock(Page::class.java) as Page<Blob>
+
+        Mockito.`when`(fakeStorage.create(Matchers.any(BlobInfo::class.java),
+                Matchers.any(ByteArray::class.java))).thenReturn(blob1).thenReturn(blob2)
+
+        Mockito.`when`(fakeStorage.list(eq(this.bucketName),
```

Comment:
The same about formatting and any:
```
        Mockito.`when`(fakeStorage.list(
            eq(this.bucketName), any(Storage.BlobListOption::class.java), any(Storage.BlobListOption::class.java)))
            .thenReturn(fakePage)
```

Please make such calls a bit more readable everywhere in these tests